.github/workflows/packaging.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Packaging - Nightly Artifacts
+name: Packaging
```

Comment:
File name is also renamed: `packaging-wheels.yml` -> `packaging.yml`

Is it OK?

Reply:
Should be ok! We'll just update a couple references (I wanted to do this anyways)

Reply:
I've renamed `pckaging-wheels.yml` in `dev/release/03-binary-sign.sh`.